portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -552,6 +552,37 @@ definition {
 			appStatus = "Active");
 	}
 
+	@description = "This test assert that user can filter app by Categories, Liferay Version, Price filters in marketplace store."
+	@priority = "3"
+	test FilterMarketplaceStoreApps {
+		property test.name.skip.portal.instance = "AppManagement#FilterMarketplaceStoreApps";
+
+		task ("Give user login marketplace") {
```

Comment:
The Give should Given.

Reply:
Will pull other two testcases in other tickets